galasa-ui/src/components/test-runs/TestRunDetails.tsx

```diff
@@ -115,7 +115,7 @@ const TestRunDetails = ({ runId, runDetailsPromise, runLogPromise, runArtifactsP
                 {translations("status")}: {run?.status}
```

Comment:
TestRunDetails has a warning in one of its useEffects, can this be resolved please?